src/converter_manager.py

```
@@ -54,26 +55,19 @@ def convert_all(self) -> None:
     def save_idf(self, output_path: Path) -> None:
         output_path.parent.mkdir(parents=True, exist_ok=True)
         self.logger.info("Saving IDF to {}...", output_path)
-        self._idf.saveas(str(output_path))
+        self._idf.save(output_path)
 
     def load_idf(self, idf_path: Path) -> None:
         self.logger.info("Loading IDF from {}...", idf_path)
```

Ruff (0.15.20)

[error] Too many arguments for `logging` format string

(PLE1205)